app/core/personutils.js

```diff
@@ -99,6 +99,9 @@ personUtils.isRemoveable = function(person) {
 personUtils.validateFormValues = function(formValues, isNameRequired, dateFormat, currentDateObj) {
   var validationErrors = {};
 
+  var INVALID_DATE_TEXT = 'Hmm, this date doesn’t look right';
```

Comment:
I think @GordyD already has tests around them in some fashion, so that might not be fully necessary, at least if the goal is to check what the text *is*.